Tools/LoadNMs.linq

```diff
@@ -0,0 +1,305 @@
+<Query Kind="Program">
+  <Reference>&lt;RuntimeDirectory&gt;\netstandard.dll</Reference>
+  <Reference>&lt;RuntimeDirectory&gt;\System.Net.Http.dll</Reference>
+  <NuGetReference>Microsoft.Identity.Client</NuGetReference>
+  <NuGetReference>Newtonsoft.Json</NuGetReference>
+  <NuGetReference>UKHO.FileShareAdminClient</NuGetReference>
+  <NuGetReference>UKHO.WeekNumberUtils</NuGetReference>
+  <Namespace>Microsoft.Identity.Client</Namespace>
+  <Namespace>System.Net.Http</Namespace>
+  <Namespace>System.Net.Http.Headers</Namespace>
+  <Namespace>System.Threading.Tasks</Namespace>
+  <Namespace>UKHO.FileShareAdminClient</Namespace>
+  <Namespace>UKHO.FileShareClient</Namespace>
+  <Namespace>System.Globalization</Namespace>
+  <Namespace>UKHO.WeekNumberUtils</Namespace>
+  <Namespace>UKHO.FileShareAdminClient.Models</Namespace>
+  <Namespace>UKHO.FileShareClient.Models</Namespace>
+</Query>
+
+const string BUSINESS_UNIT = "MaritimeSafetyInformation";
+const string PRODUCT_TYPE_ATTRIBUTE_VALUE = "Notices to Mariners";
+
+private readonly Func<DateTime, DateTime> WEEKLY_NMS_RETENTION_PERIOD = d => d.AddDays(800);
+private readonly Func<DateTime, DateTime> DAILY_NMS_RETENTION_PERIOD = d => d.AddDays(15);
+
+private readonly bool REPLACE_ALL_DATA_IN_BUSINESS_UNIT = false;
+
+private async Task Main()
+{
+	var configLive = new
+	{
+		BaseAddress = @"https://files.admiralty.co.uk/",
+		ClientId = "30cc156d-765f-43ea-8c84-a97d5b8ce852",
```

Comment:
I personally do not feel comfortable making these details public. Fair enough if these were B2C app registrations that are already in the public domain, but these are from the internal domain.

Reply:
I completely agree, so have deferred to storing this file in the "Support Engineer Toolbox".

Please see https://github.com/UKHO/supportengineer-toolbox/pull/62